packages/design-system/src/components/N8nButton/Button.vue

```diff
@@ -444,4 +454,9 @@ $loading-overlay-background-color: rgba(255, 255, 255, 0);
 .float-right {
 	float: right;
 }
+
+.square {
```

Comment:
is this class needed? are not we centering by default?

Reply:
actually not, in the case of icon only button the shape is already square but the icon is not centered (icon has the same position as if the button contained text as well)

but replacing `.square` with `&:not(.text)` and putting it under `.icon` is having the same effect but the `square` prop and class name are still necessary for other reasons already mentioned